src/cli/cli.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/f0o/turbo-pancake/common"
+	"github.com/f0o/turbo-pancake/utils"
+	"github.com/f0o/turbo-pancake/worker"
+)
+
+//spawnWorkers : Wrapper that creates Worker-Goroutines
+func spawnWorkers() {
+	for w := 0; w < *common.Threads; w++ {
+		uuid, _ := utils.UUIDv4()
+		utils.Debug("test", "%v", uuid)
+		go worker.Worker(uuid[:])
+	}
+}
+
+//getReturns : Main Loop to wait and process ReturnSpec from the ReturnQueue
+func getReturns() {
+	for ret := range common.ReturnQueue {
+		common.JobLength--
+		if ret.Exit > 0 {
+			utils.Debug(ret.ID, "Worker '%s' Exited with Code '%d'", ret.Worker, ret.Exit)
+		}
+		utils.Debug(ret.ID, "Returned: %v", ret.Return)
+		fmt.Printf(*common.OutFormat, ret.Return)
+		if common.JobLength == 0 {
+			break
+		}
+	}
+}
+
+//createQueues : Creates Queues from Input
+func createQueues() {
+	common.JobQueue = make(chan common.JobSpec, len(common.Input))
+	common.ReturnQueue = make(chan common.ReturnSpec, len(common.Input))
+}
+
+//createJobs : Creates JobSpecs
+func createJobs() {
+	for i := 0; i < len(common.Input); i++ {
+		uuid, _ := utils.UUIDv4()

```

Comment:
The only line that calls 
utils.UUIDv4()

but only is using single return in a double return context. maybe second return should be removed or checked for errors

Reply:
error return is implemented for future use - all of this unstable WIP